static/jscn/fragment.js

```diff
@@ -59,22 +59,32 @@ async function populate_datetxt_list(channel) {
         "rqstdata": "listdate",
         "channame": channel,
     }, function (data) {
-        for (let indx in data) {
+
+        const dataSorted = Object.entries(data);
+        dataSorted.reverse();
+
+        for (let i = 0; i < dataSorted.length; i++) {
+
+            const element = dataSorted[i];
+            const date = element[0];
+            const url = element[1];
+
```

Comment:
Could you please do away with excess in-line spacings and set the indentation right?